src/bin/paper-init.ts

```diff
@@ -27,6 +27,11 @@ const question = [{
 ;(async() => {
   const { folder } = await inquirer.prompt(promps)
   
+  if (/\s+|\\|\/|\^|\|/.test(folder)) {
+    console.log(`\n${chalk.yellow(folder)} contains illegal characters.`)
```

Comment:
contains unsupported spaces

Reply:
Is it necessary to explain what characters are illegal?

Reply:
agree

Reply:
done